src/ape/api/providers.py

```diff
@@ -1278,7 +1278,7 @@ def prepare_transaction(self, txn: TransactionAPI) -> TransactionAPI:
                 txn.max_priority_fee = self.priority_fee
 
             if txn.max_fee is None:
-                txn.max_fee = self.base_fee + txn.max_priority_fee
+                txn.max_fee = self.base_fee * 2 + txn.max_priority_fee
```

Comment:
some providers are using the eth fee history API now for base fee, and some aren't.
Does it matter here? I just want to make sure you know that.

Like sometimes that API doesnt exist, like in eth tester, so it doesnt use that. But some providers do use it.